### PR TITLE
[v6r21] moved accounting clients to use Client as base

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*Accounting
CHANGE: accounting clients use DIRAC.Core.Base.Client as base

ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
@atsareg can you check this failing test? https://travis-ci.org/DIRACGrid/DIRAC/jobs/520777025#L1560

File: requirements.txt [hunk 1]
```diff
@@ -5,10 +5,10 @@ git+https://github.com/DIRACGrid/pyGSI.git#egg=GSI
 # From pypi
 autopep8==1.3.3
 certifi
-CMRESHandler>=1.0.0b4
 coverage
 codecov
-elasticsearch-dsl>=2.0.0
+elasticsearch-dsl~=6.3.1
+CMRESHandler>=1.0.0b4
```
Comment (fstagni) [inline requirements.txt]: @atsareg @andresailer I think these changes fix the issues that Travis show on other PRs, like those seen in https://travis-ci.org/DIRACGrid/DIRAC/jobs/520371438

File: AccountingSystem/Client/DataStoreClient.py [hunk 2]
```diff
@@ -1,86 +1,81 @@
...
+
+class DataStoreClient(Client):
   """
     Class providing front end access to DIRAC Accounting DataStore Service
      - It allows to reduce the interactions with the server by building and list of
     pending Registers to be sent that are sent in a bundle using the commit method.
      - In case the DataStore is down Registers are sent as DISET requests.
   """
-  def __init__( self, setup = False, retryGraceTime = 0 ):
-    self.__setup = setup
```
Comment (chaen) [inline AccountingSystem/Client/DataStoreClient.py]: What was this setup thing ? is it not needed anymore ?

File: AccountingSystem/Client/DataStoreClient.py [hunk 3]
```diff
@@ -1,86 +1,81 @@
...
+
+  def __init__(self, retryGraceTime=0, **kwargs):
+    """ Simple constructor
```
Comment (chaen) [inline AccountingSystem/Client/DataStoreClient.py]: Docstring for retryGraceTime ?

File: AccountingSystem/private/Plotters/BaseReporter.py [hunk 4]
```diff
@@ -21,291 +28,374 @@ class BaseReporter( DBUtils ):
...
+          ('kHS06',
+           1000,
+           750),
+          ('MHS06',
+           10 ** 6,
+           1)),
+      'bytes': (
+          ('MB / s',
+           10 ** 6,
+           1000),
```
Comment (chaen) [inline AccountingSystem/private/Plotters/BaseReporter.py]: GRRRRRRRRRRRRR here it is power of 10 again. Shall we change that too ?

File: AccountingSystem/Client/DataStoreClient.py (hunk 2 above)
Comment (fstagni) [reply AccountingSystem/Client/DataStoreClient.py]: Something very (very!) old that's certainly not needed anymore.

File: AccountingSystem/private/Plotters/BaseReporter.py (hunk 4 above)
Comment (fstagni) [reply AccountingSystem/private/Plotters/BaseReporter.py]: Just tell me what to put... I am a bit lost right now at what's the unit to use :)

Comment (fstagni) [reply AccountingSystem/private/Plotters/BaseReporter.py]: @chaen ...?

Comment (chaen) [reply AccountingSystem/private/Plotters/BaseReporter.py]: Put 1024 here for sizes, probably the easiest. Or use the utility function, but maybe an overkill

**Comment by atsareg** [comment]
@fstagni : the failing PoolComputingElement test is not failing if run manually. This can be some timing hick-up. Rerunning the test will be probably OK

**Comment by fstagni** [comment]
Review OK?

File: AccountingSystem/private/Plotters/BaseReporter.py (hunk 4 above)
Comment (andresailer) [reply AccountingSystem/private/Plotters/BaseReporter.py]: Shouldn't the powers of 10 be changed to corresponding powers of 2? What does the 1000 represent anyway?

**Pull request merged**